src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/LogManagerConfigTest.java

```diff
@@ -463,12 +464,15 @@ void GIVEN_runtimeConfiguration_withOldProcessingFileFormat_WHEN_onStartUp_THEN_
                 PERSISTED_COMPONENT_CURRENT_PROCESSING_FILE_INFORMATION,
                 componentName);
 
+        long now = Instant.now().toEpochMilli();
```

Comment:
This is the only thing that changed on this file the rest are just auto formatting.